README.md

```diff
@@ -27,40 +27,40 @@ The recommended way of obtaining this tool is installing it as a [.NET Tool](htt
 #### Global Tool
 
 - `dotnet tool install -g Kontent.Ai.ModelGenerator`
-- `KontentModelGenerator --projectid "<projectid>" [--namespace "<custom-namespace>"] [--outputdir "<output-directory>"] [--withtypeprovider <True|False>] [--structuredmodel <True|False>] [--filenamesuffix "<suffix>"]`
+- `KontentModelGenerator --projectid "<projectid>" [--namespace "<custom-namespace>"] [--outputdir "<output-directory>"] [--withtypeprovider <True|False>] [--structuredmodel <"{structured_model}"] [--filenamesuffix "<suffix>"]`
```

Comment:
unnecessary quotation marks? You didn't use them in https://github.com/kontent-ai/delivery-sdk-net/pull/361/files

And missing '>'. The same applies to examples bellow